scripts/zones/Boneyard_Gully/bcnms/like_the_wind.lua

```diff
@@ -35,9 +35,9 @@ end;
 function onBcnmLeave(player,instance,leavecode)
 
     if (leavecode == 2) then -- play end CS. Need time and battle id for record keeping + storage
-        player:startEvent(32001,1,1,1,instance:getTimeInside(),1,0,0);
+        player:startEvent(32002,1,1,1,instance:getTimeInside(),1,0,0);    
     elseif (leavecode == 4) then
-        player:startEvent(32002);
+        player:startEvent(32003);
```

Comment:
`32003` appears to be the exit menu. Don't think this is right.

![image](https://user-images.githubusercontent.com/17911103/42660586-ab865ed8-85f9-11e8-9043-6b660dcefd03.png)